solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrClientBase.java

```diff
@@ -121,7 +121,7 @@ protected ResponseParser responseParser(SolrRequest<?> solrRequest) {
     return solrRequest.getResponseParser() == null ? this.parser : solrRequest.getResponseParser();
   }
 
-  protected ModifiableSolrParams initalizeSolrParams(
+  protected ModifiableSolrParams initializeSolrParams(
```

Comment:
This one is prorected, meaning that potentially 3rd party sub classes could call it? However, how common would that be?

Reply:
Dunno...   Would a 3rd party sub class immediately break and then you fix it?  I could easily do the deprecation however if you think that is just the easier process.....   Especailly for backporting...

Reply:
I pushed the original method signature as deprecated. That deprecation can be removed in a followup commit to main after backporting. Or we could just do the plain rename and add in the deprecation when backporting to 9x?

Reply:
i prefer to add the deprecation, and then do the main, branch_9x, and then come back and in a seperate commit remove the deprecated code.   Just to make it simpler to backport.   However, that might be me..  I've struggled on some of the backporting!

Reply:
okay, in looking I see how you did it...!  All makes sense..      We need a plan for removing deprecated code in `main` anyway, there is so much of it!